api/v1/views/__init__.py

```diff
@@ -0,0 +1,3 @@
+from flask import Blueprint
+app_views = Blueprint("app_views", __name__, url_prefix="/api/v1")
+from api.v1.views.index import *
```

Comment:
We should move this to line 1, so the imports will be alphabetical

Reply:
The order of import is important as we need to create the Blueprint and import that package. We need to leave it as is